src/tests/unit/tests/background/global-action-creators/global-action-creator.test.ts

```diff
@@ -217,7 +160,7 @@ class GlobalActionCreatorValidator {
         commandActions: this.commandActionsContainerMock.object,
         featureFlagActions: this.featureFlagActionsContainerMock.object,
         launchPanelStateActions: this.launchPanelStateActionsContainerMock.object,
-        scopingActions: this.scopingActionsContainerMock.object,
+        scopingActions: null,
```

Comment:
Can we get rid of this?

Reply:
Not without casting. I can go either way, let me know